src/main/scala/booksImplementation/BooksDetails.scala

```diff
@@ -0,0 +1,49 @@
+package booksImplementation
```

Comment:
please update the package name to make it more meaningful

Reply:
renamed the package as dao